modules/help_editor/php/helpfile.class.inc

```diff
@@ -106,7 +106,7 @@ class HelpFile
         $DB =& \Database::singleton();
 
         // update the help file
-        $success = $DB->unsafeupdate('help', $set, array('helpID' => $this->helpID));
+        $success = $DB->update('help', $set, array('helpID' => $this->helpID));
```

Comment:
there is also an unsafeinsert on line 90 that we could change as part of this PR

Reply:
Nice catch! I'll change that now

Reply:
so i think you're all good! i'm not sure why we changed it to unsafe insert/update in the first place

Reply:
@zaliqarosli out of curiousity, why is the insert function `static public` but the update function is not?

Reply:
@ZainVirani i have no idea, i wasn't the original author